ONSdigital/dp-import
# Add Cantabular export schemas and events for consistency with import.…

… Upgrade to dp-kafka v3

### What

- Add Cantabular export pipeline Avro schemas and events, to be consistent with the import pipeline, and have all the definitions centralised in a single place.
- Upgrade to dp-kafka/v3, and go v 1.17

### How to review

- Check that export-start schema and event match the current values:
  - schema: https://github.com/ONSdigital/dp-cantabular-csv-exporter/blob/develop/schema/schema.go#L19
  - event: https://github.com/ONSdigital/dp-cantabular-csv-exporter/blob/develop/event/event.go#L4

- Check that csv-created schema and event match the current values:
  - schema: https://github.com/ONSdigital/dp-cantabular-csv-exporter/blob/develop/schema/schema.go#L36
  - event: https://github.com/ONSdigital/dp-cantabular-csv-exporter/blob/develop/event/event.go#L12

- Check that csvw-created schema and event match the current values:
  - schema: https://github.com/ONSdigital/dp-cantabular-metadata-exporter/blob/develop/schema/schema.go#L38
  - event: https://github.com/ONSdigital/dp-cantabular-metadata-exporter/blob/develop/event/event.go#L13


### Who can review

anyone
